ciao-storage/ceph.go

```diff
@@ -166,16 +170,22 @@ func (d CephDriver) MapVolumeToNode(volumeUUID string) (string, error) {
 // UnmapVolumeFromNode unmaps a ceph volume from a local device on a node.
 func (d CephDriver) UnmapVolumeFromNode(volumeUUID string) error {
 	args := append(d.getCredentials(), "unmap", volumeUUID)
-	return exec.Command("rbd", args...).Run()
+	cmd := exec.Command("rbd", args...)
+
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("Error when running: %v: %v: %s", cmd.Args, err, out)
+	}
+	return nil
 }
 
 // GetVolumeMapping returns a map of volumeUUID to mapped devices.
 func (d CephDriver) GetVolumeMapping() (map[string][]string, error) {
 	args := append(d.getCredentials(), "showmapped", "--format", "json")
 	cmd := exec.Command("rbd", args...)
-	data, err := cmd.Output()
+	data, err := cmd.CombinedOutput()
```

Comment:
Ah @markdryan  that's a nice solution. I'd have hoped that rbd did the right right of not writing random stuff to stderr with a zero exit code. But this fix is the safest.